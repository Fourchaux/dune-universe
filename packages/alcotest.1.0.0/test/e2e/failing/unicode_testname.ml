let () =
  Alcotest.run "suite-name"
    [ ("ðŸ”¥", [ Alcotest.test_case "First test case" `Quick (fun () -> ()) ]) ]
