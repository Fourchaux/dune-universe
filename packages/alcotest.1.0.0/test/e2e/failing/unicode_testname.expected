Error: "\240\159\148\165" is not a valid test label (must match ^[a-zA-Z0-9_- ]+$).
